draft-ietf-homenet-front-end-naming-delegation.md

```diff
@@ -435,8 +439,10 @@ The information provided by the DM to the HNA is retrieved by the HNA with an AX
 AXFR enables the response to contain any type of RRsets.
 
 To retrieve the necessary information to build the Public Homenet Zone, the HNA MUST send a DNS request of type AXFR associated with the Registered Homenet Domain.
-The DM MUST respond with a zone template.
-The zone template MUST contain a RRset of type SOA, one or multiple RRset of type NS and zero or more RRset of type A or AAAA.
+
+The zone that is returned by the DM is used by the HNA as a template to build its own zone.
+
+The zone template MUST contain a RRset of type SOA, one or multiple RRset of type NS and zero or more RRset of type A or AAAA (if the NS are in-bailiwick {{!RFC8499}}).
```

Comment:
I am wondering if we should add the TTL values accepted by the DOI.  This is clarified in the #zonetemplate section, so that is not essential.

Reply:
The TTL values would be part of any RR received in the template, but maybe you mean, that they should be taken as a recommendation?

Reply:
Yes I meant to take it as a recommendation.